cluster/ci/config/prow/config.yaml

```diff
@@ -73,6 +73,7 @@ tide:
     - openshift/origin-web-console
     - openshift/origin-web-console-server
     - openshift/cluster-operator
+    - openshift/service-catalog
```

Comment:
We are running no jobs on this repo, so let's not use tide on it.

Reply:
Ok, but in the future we might? I'll remove it for now.